src/Range.jsx

```diff
@@ -219,7 +218,7 @@ class Range extends React.Component {
     const diffToNext = direction * (bounds[nextHandle] - value);
     if (!this.pushHandle(bounds, nextHandle, direction, threshold - diffToNext)) {
       // revert to original value if pushing is impossible
-      bounds[handle] = originalValue;
+      bounds[handle] = bounds[nextHandle] - (direction * threshold);
```

Comment:
`pushable` may be a boolean

Reply:
yes, there is implicit-conversion when calculate. But of course, it's not serious. I will change to explicit.